app/javascript/components/visual-settings-form/expression-editor/index.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { useState } from 'react';
+import { Button, Dropdown, Select, SelectItem } from 'carbon-components-react';
+import './expression-editor-style.css';
+
+const EditExpression = () => {
+  const newElement = "<new element>";
+  const items = [
```

Comment:
please move all these `const` values to a helper file

Reply:
made the changes